cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -0,0 +1,70 @@
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
```

Comment:
Can you add the comments above this block that are in the other files explaining what it's used for

Reply:
This is not done